packages/search-ui-elasticsearch-connector/src/handlers/common.ts

```diff
@@ -22,7 +22,8 @@ export function fieldResponseMapper(
     {
       id: { raw: item.id },
       _meta: {
-        id: item.rawHit._id
+        id: item.rawHit._id,
+        raw: item.rawHit
```

Comment:
This causes `isFieldValueWrapper` to return `true` for `_meta`, 
https://github.com/elastic/search-ui/blob/17488c45f21a496ad5cfbe1e09e624240647b93b/packages/react-search-ui-views/src/Result.tsx#L32-L38
which means we get `"_meta": [object Object]` in the results.

Reply:
This can be solved by adding a type check:
```typescript
function isFieldValueWrapper(object) {
  return (
    object &&
    (Object.prototype.hasOwnProperty.call(object, "raw") && typeof object["raw"] != "object" ||
      Object.prototype.hasOwnProperty.call(object, "snippet"))
  );
}
```

or simply renaming the property to something like `raw_es_hit`. What do you think is the best solution?
Also, it might be worth to write a test for this.

Reply:
Done.